src/drive/web/modules/move/MoveModal.jsx

```diff
@@ -152,7 +152,8 @@ export class MoveModal extends React.Component {
           $ne: TRASH_DIR_ID
         }
       })
-      .sortBy([{ type: 'asc' }, { name: 'asc' }])
+      .indexFields(['dir_id', 'type', 'name'])
+      .sortBy([{ dir_id: 'asc' }, { type: 'asc' }, { name: 'asc' }])
```

Comment:
Do we need the new sort option here?

Reply:
Yes because we need to sort by the same fields that the ones indexed.

Reply:
Uh, ok.